TemplateApplication.xcodeproj/project.pbxproj

```diff
@@ -1194,23 +1167,38 @@
 				minimumVersion = 0.3.3;
 			};
 		};
+		970563142A532BFC005BB074 /* XCRemoteSwiftPackageReference "SpeziHealthKit" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/StanfordSpezi/SpeziHealthKit";
+			requirement = {
+				branch = "feature/determine-health-authorizations-given";
+				kind = branch;
+			};
+		};
+		970563172A532C6A005BB074 /* XCRemoteSwiftPackageReference "SpeziScheduler" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/StanfordSpezi/SpeziScheduler";
+			requirement = {
+				branch = "feat/flag-determining-notification-auth";
+				kind = branch;
+			};
+		};
+		9705631A2A532C87005BB074 /* XCRemoteSwiftPackageReference "SpeziOnboarding" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/StanfordSpezi/SpeziOnboarding";
+			requirement = {
+				branch = "feat/allow-async-action-OnboardingActionsView";
+				kind = branch;
+			};
+		};
```

Comment:
Good for now, let's just make sure that we don't forget about this before we merge the PR 👍

Reply:
Sure, I just included it so that everyone can test the application without the need to manually download specific versions of the dependencies.